doc/Installation/Installation-(RHEL-CentOS).md

```diff
@@ -20,15 +20,15 @@ service mysqld start
 **MariaDB**
 ```bash
 yum install net-snmp mariadb-server mariadb-client
-chkconfig mariadb on
-service mariadb start
+systemctl enable mariadb
+systemctl start mariadb
```

Comment:
These two wont work for CentOS 6 MariaDB. Retain `chkconfig` and `service` they will work with CentOS 7 as well afaik.